pkg/ui/react-app/src/thanos/pages/blocks/helpers.ts

```diff
@@ -146,3 +146,21 @@ export const getBlocksByCompactionLevel = (blocks: Block[], compactionLevel: num
   const blockResult = blocks.filter((block) => block.compaction.level === compactionLevel);
   return blockResult;
 };
+
+export const getFilteredBlockPools = (
```

Comment:
Maybe we could add a unit test or two for this?

Reply:
Yeah for sure, I can add one

Reply:
Hey @GiedriusS you can review now